datasets/swiss_judgment_prediction/swiss_judgment_prediction.py

```diff
@@ -57,37 +65,45 @@ def __init__(self, language: str, languages=None, **kwargs):
         """BuilderConfig for SwissJudgmentPrediction.
 
         Args:
-        language: One of de,fr,it, or all_languages
+        language: One of de, fr, it, or all, or all+mt
           **kwargs: keyword arguments forwarded to super.
         """
         super(SwissJudgmentPredictionConfig, self).__init__(**kwargs)
         self.language = language
-        if language != "all_languages":
-            self.languages = [language]
-        else:
+        if language == "all":
+            self.languages = languages if languages is not None else _ORIGINAL_LANGUAGES
+        elif language == "all+mt":
             self.languages = languages if languages is not None else _LANGUAGES
+        else:
```

Comment:
I just noticed the `languages` parameter. What is your intention with this?

Reply:
I mean, I can imagine one purpose, but then this implementation is not aligned with it...

Reply:
Hmm, good point. Probably some sort of relic. I removed it. Thanks